bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/Handle.java

```diff
@@ -21,7 +21,7 @@
 package org.apache.bookkeeper.client.api;
 
 import java.util.concurrent.CompletableFuture;
-import lombok.SneakyThrows;
+import org.apache.bookkeeper.client.impl.BKExceptionHandler;
```

Comment:
Make BKExceptionHandler.HANDLER as part of BKException. rather than having a separate class. this also prevent api package referencing impl package, which allows producing a clean api package.